odoo/addons/base/models/ir_model.py

```diff
@@ -855,8 +855,13 @@ def _reflect_model(self, model):
                 keys = [key for key in new_vals if old_vals[key] != new_vals[key]]
                 self.pool.post_init(record.modified, keys)
                 old_vals.update(new_vals)
-            if module and not field.manual and (module == model._original_module or module in field._modules):
-                to_xmlids.append(name)
+            if module and (module == model._original_module or module in field._modules):
+                # remove this and only keep the else clause if version >= saas-12.4
```

Comment:
@mart-e see this

Reply:
@mreficent I know but it is not a problem to keep it. We wrote "saas-12.4" to avoid writing an unclear "master". It will be removed once we write a migration script. Thanks for the check though.